tests/infra_test.go

```diff
@@ -1106,6 +1106,22 @@ var _ = Describe("[Serial][owner:@sig-compute]Infrastructure", func() {
 				Expect(found).To(Equal(false), "Node can't contain label "+v1.CPUModelLabel+obsoleteModel)
 			})
 
+			It("should update node with new cpu model vendor label", func() {
+				nodes, err := virtClient.CoreV1().Nodes().List(context.Background(), metav1.ListOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				found := false
+				for _, node := range nodes.Items {
+					for key := range node.Labels {
+						if strings.HasPrefix(key, v1.CPUModelVendorLabel) {
+							found = true
+							break
+						}
+					}
+				}
+
+				Expect(found).To(Equal(true), "Node should contain label "+v1.CPUModelVendorLabel)
```

Comment:
Should this test check that all nodes have the label `CPUModelVendorLabel`? Currently it only checks if at least one node has the label.

Reply:
virt-handler doesn't have to be scheduled on all nodes, so there can be a case when only e.g. 1 node from 3 will have vendor label.

Reply:
Ok, then can you make it more obvious, that it is enough if one node has the label. For example:
```
for _, node := range nodes.Items {
    for key := range node.Labels {
        if strings.HasPrefix(key, v1.CPUModelVendorLabel) {
            return
        }
    }
}

Fail("No node contains label "+v1.CPUModelVendorLabel)
```